<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserClientinfoWidget</class>
 <widget class="QWidget" name="UserClientinfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Profile</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="picture_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>250</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" rowspan="2">
              <layout class="QFormLayout" name="formLayout_2">
               <property name="labelAlignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
               <item row="0" column="0">
                <layout class="QFormLayout" name="formLayout_3"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="font">
                  <font>
                   <family>Abadi MT Condensed Extra Bold</family>
                   <pointsize>24</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>ID Usuario:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="IdUser_label">
                 <property name="font">
                  <font>
                   <family>Abadi MT Condensed Extra Bold</family>
                   <pointsize>24</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <family>Abadi MT Condensed Extra Bold</family>
                   <pointsize>24</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>User name:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="UserName_label">
                 <property name="font">
                  <font>
                   <family>Abadi MT Condensed Extra Bold</family>
                   <pointsize>24</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="Email_label">
                 <property name="font">
                  <font>
                   <family>Abadi MT Condensed Extra Bold</family>
                   <pointsize>24</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <family>Abadi MT Condensed Extra Bold</family>
                   <pointsize>24</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Plan:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLabel" name="Plan_label">
                 <property name="font">
                  <font>
                   <family>Abadi MT Condensed Extra Bold</family>
                   <pointsize>24</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <family>Abadi MT Condensed Extra Bold</family>
                   <pointsize>24</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Last Log:</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QLabel" name="lastLogging_label">
                 <property name="font">
                  <font>
                   <family>Abadi MT Condensed Extra Bold</family>
                   <pointsize>18</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="font">
                  <font>
                   <family>Abadi MT Condensed Extra Bold</family>
                   <pointsize>24</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>User Since:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLabel" name="user_since_label">
                 <property name="font">
                  <font>
                   <family>Abadi MT Condensed Extra Bold</family>
                   <pointsize>18</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <family>Abadi MT Condensed Extra Bold</family>
                   <pointsize>24</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Email:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="newPhoto">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset theme="QIcon::ThemeIcon::CameraPhoto"/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
